docker-mysql-base.yml

```diff
@@ -1,8 +1,9 @@
+# https://docs.docker.com/compose/extends/#adding-and-overriding-configuration
+
 version: "3.1"
```

Comment:
3.7 here too right?

Reply:
Yup, it would be overwritten, but for the sake of consistency: a686430b5bb1bbbbaa3c1bdf41dfb7f46ef9b1ba